app/main.py

```
@@ -1 +1,46 @@
-#  write your code here
+class Animal:
+    def __init__(
+        self,
+        name: str,
+        appetite: int,
+        is_hungry: bool = True
+    ) -> None:
+
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> None:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry : bool = True) -> None:
+        super().__init__(name, 3, is_hungry)
+
+    @staticmethod
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name: str, is_hungry : None = True) -> None:
```

Dog.__init__ has an incorrect type annotation: `is_hungry : None = True`. The `is_hungry` parameter should be a boolean with default True per the task description. Change the annotation to `is_hungry: bool = True`. Functionality is currently okay (default True), but the annotation is misleading and should be corrected to match the requirement .